versioned_docs/version-2.9/how-to-guides/new-user-guides/helm-charts-in-rancher/oci-repositories.md

```diff
@@ -0,0 +1,113 @@
+---
+title: Using OCI-Based Helm Chart Repositories
+---
+
+<head>
+  <link rel="canonical" href="https://ranchermanager.docs.rancher.com/how-to-guides/new-user-guides/helm-charts-in-rancher/oci-registries"/>
+</head>
+
+:::caution
+
+This feature is experimental.
```

Comment:
Considering [our own team's uncertainty](https://github.com/rancher/rancher-docs/pull/1171#discussion_r1674511021) around the term, we should probably expand what it means to be experimental similar to https://ranchermanager.docs.rancher.com/how-to-guides/advanced-user-guides/enable-experimental-features/rancher-on-arm64 (although the recommendation may be excessive for this scenario).

Reply:
Maybe just "[x] is currently an experimental feature and is not yet officially supported in Rancher. "

Reply:
> Maybe just "[x] is currently an experimental feature and is not yet officially supported in Rancher. "

I think that's fine if we drop the `yet` from it.